collection-scripts/gather_bm_network_logs

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+. gather_service_logs_util
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/bm_network_logs"}
+SERVICE_LOG_PATH="${NETWORK_LOG_PATH}"
+
+mkdir -p "${NETWORK_LOG_PATH}"/
+
+function gather_bm_network_logs {
+  NODE=${1}
+  mkdir -p "${NETWORK_LOG_PATH}/${NODE}"
+  HOST_POD=$(oc debug --to-namespace="${MG_NAMESPACE}" node/"${NODE}" -o jsonpath='{.metadata.name}')
+  oc debug --to-namespace="${MG_NAMESPACE}" node/"${NODE}" -- /bin/bash -c \
+    'trap : TERM INT; sleep infinity & wait' > /dev/null 2>&1 &
+
+  # wait for pod creation
+  sleep 1
+
+  oc wait -n="${MG_NAMESPACE}" --for=condition=Ready pod/"${HOST_POD}" --timeout=60s
+  if [ $? -ne 0 ]; then
+    echo "ERROR: Timeout waiting for pod to become Ready"
+    oc delete pod -n="${MG_NAMESPACE}" "${HOST_POD}"
+    return
+  fi
+
+  oc exec -n="${MG_NAMESPACE}" "${HOST_POD}" -- bash -c "ip addr" > "${NETWORK_LOG_PATH}/${NODE}/ip-addr.txt"
+  oc exec -n="${MG_NAMESPACE}" "${HOST_POD}" -- bash -c "ip route" > "${NETWORK_LOG_PATH}/${NODE}/ip-route.txt"
+  oc exec -n="${MG_NAMESPACE}" "${HOST_POD}" -- bash -c "hostname" > "${NETWORK_LOG_PATH}/${NODE}/hostname.txt"
+  HOST_COMMAND='for c in $(nmcli -f uuid con show | tail -n +2); do nmcli con show $c; done'
+  oc exec -n="${MG_NAMESPACE}" "${HOST_POD}" -- bash -c \
+    "chroot /host /bin/bash -c '${HOST_COMMAND}'" \ > "${NETWORK_LOG_PATH}/${NODE}/nmcli.txt"
+
+  oc cp -n="${MG_NAMESPACE}" "${HOST_POD}":/host/etc/resolv.conf "${NETWORK_LOG_PATH}/${NODE}/resolv.conf"
+
+  oc delete pod -n="${MG_NAMESPACE}" "${HOST_POD}"
+}
+
+PIDS=()
+NODES=(${@:-$(oc get nodes -o jsonpath='{.items[?(@.status.nodeInfo.operatingSystem=="linux")].metadata.name}')})
+
+echo "WARNING: Collecting network logs on ALL linux nodes in your cluster. This could take a long time."
```

Comment:
Should this only be shown with default argument are used?

Reply:
Not sure I get it, what kind of default arguments you mean?

Reply:
When no node list is provided.